### PR TITLE
Updates to OIE sdk mapping for nodejs

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Fix broken links and remove UID from github references for nodejs mappings
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

**Comment by brianduffield-okta** [comment]
LGTM! Thanks, Van.

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/main/index.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ curl --location --request POST 'https://duffield.oktapreview.com/api/v1/sessions
 }
 ```
 
-If your application code implements these API calls and handles the responses shown, you need to update your code to use the Identity Engine SDK `idx.authenticate` method. This method encapsulates the authentication flow using recursive calls to the Identity Engine SDK method, and a successful response returns with access and ID tokens.
+If your application code implements these API calls and handles the responses shown, you need to update your code to use Identity Engine SDK methods. These methods encapsulate the authentication flow using recursive calls to the Identity Engine, and a successful response returns with access and ID tokens.
 
 See [Okta Identity Engine SDK authentication flow](#okta-identity-engine-sdk-authentication-flow)
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/main/index.md:175]:
```suggestion
See [Okta Identity Engine SDK authentication flow](#okta-identity-engine-sdk-authentication-flow).
```

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/main/index.md:175]: (sorry, I still don't understand where this link is pointing)

**Review by brentschaus-okta — APPROVED** [review]
LGTM (one nit. one comment.)

**Pull request merged**